wdl/ProcessNanoporeRNASamples.wdl

```diff
@@ -0,0 +1,77 @@
+version 1.0
+
+workflow ProcessNanoporeRNASamples {
+    input {
+        File bam
+        File bai
+        String platform
```

Comment:
On platform, I [hope](https://github.com/openwdl/wdl/issues/139).

Reply:
Are enums in WDL a thing now, or still a requested but unimplemented feature?

Reply:
unimplemented. that's what I meant by "hope" ....